core/src/test/java/org/infinispan/commons/util/concurrent/jdk8backported/BoundedEquivalentConcurrentHashMapV8BaseTest.java

```diff
@@ -64,6 +48,22 @@ public void testByteArrayPutIfAbsentFail() {
       byteArrayPutIfAbsentFail(createComparingConcurrentMap(), true);
    }
 
+   public void testSettingMaxSize() {
+      //given
+      BoundedEquivalentConcurrentHashMapV8<String, String> map = createMap(3, Eviction.LIRS);
```

Comment:
You should use ```createMap(3, evictionPolicy());``` so this way it has coverage for all of various eviction policies.